src/App/ComponentsDocumentation/components/Badge/index.js

```diff
@@ -2,28 +2,40 @@ import React from "react";
 
 import { ComponentPreview, DocContainer } from "@docutils";
 import CodeTags from "@components/CodeTags";
+import {badgeShowcase} from "./constants"
 
 const Overview = () => (
     <>
         <h2 id="overview">Overview</h2>
-        <p>Use class <CodeTags type="secondary" code=".badge" /> along with <CodeTags type="secondary" code=".badge-{color}" /> to display the badge of your choosing.</p>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <p>Badge <span className="badge badge-default">Default</span></p>
-            <p>Badge <span className="badge badge-inactive">Inactive</span></p>
-            <p>Badge <span className="badge badge-warning">Warning</span></p>
-        </ComponentPreview>
-    </>
-);
+        <p>Use class <CodeTags type="secondary" code=".badge" /> along with <CodeTags type="secondary" code=".badge-{color}" /> to display the badge of your choosing. Badges are often used with numbers. Use <CodeTags type="secondary" code=".badge-number" /> when a badge suited for use with numbers is desired.</p>
+        <ComponentPreview language="html" showCasePanel showCasePanelAdvanced={badgeShowcase} codeFigure removeOuterTag/>
+        <h3>Things to consider</h3>
+        <ul className="list list-bullet">
+            <li>A normal badge does not feature interaction, use the button component instead.</li>
+            <li>If you want removable badges considering using the tag component instead.</li>
+            <li>Do not use too long labels or whole sentences.</li>
+            <li>If possible, try not to use more than three words.</li>
+        </ul>
 
-const BadgeNumber = () => (
-    <>
-        <h2 id="badgenumber">Badges with numbers</h2>
-        <p>Badges are often used with numbers. Use <CodeTags type="secondary" code=".badge-number" /> when a badge suited for use with numbers is desired.</p>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <p>Badge <span className="badge badge-default badge-number">1</span></p>
-            <p>Badge <span className="badge badge-inactive badge-number">8</span></p>
-            <p>Badge <span className="badge badge-warning badge-number">100</span></p>
-        </ComponentPreview>
+        <h2>Do's and Dont's</h2>
```

Comment:
Use h3 here
And something is weird when rendering the ' ´s, don't know what it is, but try out another one. I.E &apos;